extensions/healthie/webhooks/messageDeleted.ts

```diff
@@ -14,23 +17,33 @@ const dataPoints = {
 
 export const messageDeleted: Webhook<
   keyof typeof dataPoints,
-  HealthieWebhookPayload
+  HealthieWebhookPayload,
+  typeof settings
 > = {
   key: 'messageDeleted',
   dataPoints,
   onWebhookReceived: async ({ payload, settings }, onSuccess, onError) => {
-    const { resource_id: deletedMessageId } = payload
-
-    if (isNil(deletedMessageId)) {
-      await onError({
-        // We should automatically send a 400 here, so no need to provide info
-      })
-    } else {
+    try {
+      const { sdk } = await createSdk({ settings })
+      const deletedMessageId = payload.resource_id.toString()
+  
+      const messageResponse = await sdk.getMessage({ id: deletedMessageId })
+      const conversationResponse = await sdk.getConversation({ id: messageResponse?.data?.note?.conversation_id })
```

Comment:
i'm not sure we'd get good information here from healthie if the messageResponse was undefined. What do you think about some validation in between the two API calls?

Reply:
Healthie errors are pretty good in this case. Example: 
API Key is Invalid. {"response".{"errors".[{"message"."API Key is Invalid"}],"status".200,"headers".{}},"request".{"query"."query getGoal($id. ID!) {\n goal(id. $id) {\n id\n user_id\n }\n}","variables".{"id"."72520"}}};